llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -859,10 +859,11 @@ static bool removeSYCLKernelsConstRefArray(GlobalVariable *GV) {
   for (auto It = IOperands.begin(); It != IOperands.end(); It++) {
```

Comment:
Shall we rename the function into something more generic than `removeSYCLKernelsConstRefArray`?

Reply:
The function does pretty specific thing and has assumptions about the nature of the object being removed.
If eventually it starts removing something else, it will will be a good moment to rename the function, now now.